src/pages/experiments/[experimentId]/plots-and-tables/volcano/index.jsx

```diff
@@ -156,26 +192,35 @@ const VolcanoPlot = () => {
     const date = moment.utc().format('YYYY-MM-DD-HH-mm-ss');
     const fileName = `de_${experimentId}_${cellSet}_vs_${compareWith}_${date}.csv`;
 
-    const disabled = plotData.length === 0 || loading || _.isEmpty(spec.spec) || error;
+    const disabled =
+      plotData.length === 0 || loading || _.isEmpty(spec.spec) || error;
 
     return (
       <CSVLink data={data} filename={fileName}>
-        <Button disabled={disabled} onClick={(e) => e.stopPropagation()} size='small'>Export as CSV...</Button>
+        <Button
+          disabled={disabled}
+          onClick={e => e.stopPropagation()}
+          size="small"
+        >
+          Export as CSV...
+        </Button>
       </CSVLink>
     );
   };
 
   if (!config) {
-    return (<Skeleton />);
+    return <Skeleton />;
   }
 
   const renderPlot = () => {
     if (error) {
       return (
         <PlatformError
-          description='Could not load differential expression data.'
+          description="Could not load differential expression data."
```

Comment:
single quotes is fine, why changing it?

Reply:
Something is wrong with your linter @StefanBabukov Can you save over all your files again so you get a consistent quote standard?